src/Basemaps/BasemapContainer.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { BasemapSliderContainer } from './styled'
+import OpenStreetMap from './OpenStreetMap'
+import BlankWhite from './BlankWhite'
+import StamenTerrain from './StamenTerrain'
+import StamenTonerDark from './StamenTonerDark'
+import StamenTonerLite from './StamenTonerLite'
+import { connectToMap } from 'Map'
+
+const LAYER_TYPE_ID = '_ol_kit_basemap'
+
+class BasemapContainer extends Component {
+  constructor (props) {
+    super(props)
+
+    this.BASEMAP_OPTIONS = [
+      <OpenStreetMap key='openStreetMap' layerTypeID={LAYER_TYPE_ID} map={props.map} />,
+      <StamenTerrain key='stamenTerrain' layerTypeID={LAYER_TYPE_ID} map={props.map} />,
+      <StamenTonerDark key='stamenTonerDark' layerTypeID={LAYER_TYPE_ID} map={props.map} />,
+      <StamenTonerLite key='stamenTonerLite' layerTypeID={LAYER_TYPE_ID} map={props.map} />,
+      <BlankWhite key='blankWhite' layerTypeID={LAYER_TYPE_ID} map={props.map} />
+    ]
+
+    this.state = {
+      showBasemaps: false,
+      basemapOptions: this.BASEMAP_OPTIONS
+    }
+  }
+
+  componentDidMount () {
+    this.props.map.on('click', () => this.setState({ showBasemaps: false }))
+  }
```

Comment:
note for CR- its not ideal that a callback is listening and performing state updates on every map click. could we attach the listener only when `showBasemaps` is set to true and listen for the off-click then remove the listener until a basemap selection is clicked again?